src/assets/icons/index.tsx

```
@@ -0,0 +1,40 @@
+import {SvgProps} from 'react-native-svg';
+import Home from '../svg/Home.svg';
+import Log from '../svg/Log.svg';
+import MyPortfolio from '../svg/MyPortfolio.svg';
+import Profile from '../svg/Profile.svg';
+
+export const enum IconNamesI {
+  home = 'Home',
+  log = 'Log',
+  profile = 'Profile',
+  myPortfolio = 'myPortfolio',
+}
```

Biome (1.9.4)

[error] The enum declaration should not be const

Const enums are not supported by bundlers and are incompatible with the 'isolatedModules' mode. Their use can lead to import inexistent values.
See TypeScript Docs for more details.
Safe fix: Turn the const enum into a regular enum.


(lint/suspicious/noConstEnum)